concourse/scripts/spec/script_testing_helpers.rb

```diff
@@ -0,0 +1,100 @@
+require "aruba/rspec"
+require "pathname"
+
+def find_git_repository_path()
+  path = __FILE__
+  begin
+    if File.exists? File.join(path, '.git')
+      return path
+    end
+    path = File.realpath(File.join(path, '..'))
+  end while path != "/"
+  raise "Cannot find a git repository parent of '#{File.dirname(__FILE__)}'"
+end
+
+def repostory_name()
+  File.basename(find_git_repository_path())
+end
+
+def prepare_aruba_config()
```

Comment:
Could this be moved to a `support` file or `spec_helper` so that we don't need to call the method in each test?

ref: https://github.com/alphagov/paas-cf/blob/62aba6b02de37f21cf957d08b6f4b7ba7248c7d4/concourse/scripts/spec/spec_helper.rb#L102